src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -132,14 +120,17 @@ export const StakingOpportunities = ({ assetId, accountId }: StakingOpportunitie
     [accountId, assetId, history],
   )
 
-  const columns = useMemo(
+  const columns: (ColumnGroup<UserStakingOpportunityWithMetadata> & {
+    Cell: Renderer<CellProps<UserStakingOpportunityWithMetadata>>
+  })[] = useMemo(
     () => [
       {
         Header: <Text translation='defi.validator' />,
         id: 'moniker',
+        // @ts-ignore this isn't a standard property of column but is somehow used by our <ReactTable /> implementation and passed down
```

Comment:
Wanted to properly type `columns` so we don't have to type assign Cell's `row.original` but our additional properties make this cancer, adding an additional intersection somehow rugs the types.

The reason this worked in the first place is because this had no typings and react-table `Column` is typed as a doubly unsafe `Column<D extends object = {}> =` (default empty object, and conventionally wrong broad `object` type)

Reply:
i am still skeptical about this, but lets get this in for now